modules/ob-relationship-of-build-run-with-tekton-tasks.adoc

```diff
@@ -0,0 +1,11 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-runs.adoc
+
+:_content-type: CONCEPT
+[id="ob-relationship-of-build-run-with-tekton-tasks_{context}"]
+= Relationship of build run with Tekton tasks
```

Comment:
I'd consider renaming this to something simpler

```suggestion
= Build run and Tekton tasks
```

Reply:
But the suggested heading does not set the correct context for this content. We are not discussing abt Build run and Tekton tasks at individual level. We are discussing abt the relationship between the two.